server/src/test/java/io/spine/server/bc/BoundedContextShould.java

```diff
@@ -337,4 +351,53 @@ public void do_not_expose_invisible_aggregate() {
         assertFalse(boundedContext.findRepository(SecretProject.class)
                                   .isPresent());
     }
+
+    @Test(expected = NullPointerException.class)
+    public void throw_NPE_when_default_state_is_null() {
```

Comment:
Perhaps this test does not belong here. Or it should be named differently. My clue is that the following sentence does not make much sense:

"BoundedContext should ... throw NPE when default state is null".

Why throw? Which default state? etc. 

I'd suggest to rename the test method and probably also add some Javadocs to it to explain what we are testing.